data/json/monsters/mammal.json

```diff
@@ -591,6 +600,7 @@
     "fear_triggers": [ "SOUND", "PLAYER_CLOSE" ],
     "placate_triggers": [ "MEAT" ],
     "death_function": [ "NORMAL" ],
+    "harvest": "animal_medium",
```

Comment:
Red fox has size "SMALL", why give it harvest animal_medium? It needs to be a new category for animal_small.

Also will need to remove the FUR/BONES/FAT flags like you've done similarly on line 422.